packages/genesys-spark-components/src/components/stable/gux-table-toolbar/gux-table-toolbar.scss

```diff
@@ -5,6 +5,10 @@
   justify-content: space-between;
   width: 100%;
   max-height: var(--gse-ui-dataTableItems-tableToolbar-height);
+  opacity: 0;
+  animation-name: show-toolbar;
+  animation-delay: 0.5s;
+  animation-fill-mode: forwards;
```

Comment:
Not a huge fan of this approach either but it is better what we have now so worth doing I think.
I do think we should move this into a mixin so that we can name it as this is a workaround and someone in the future will wonder why it was added.